package.json

```diff
@@ -30,7 +30,8 @@
     "flow",
     "react.js",
     "reactNative.js",
-    ".flowconfig"
+    ".flowconfig",
+    ".yalc"
```

Comment:
We don't need the `.yalc` trick anymore, we can ask Live on another PR

Reply:
yeah please let's not ship .yalc in NPM ^^